.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Data request
+about: Suggest data to add to this platform
+
+---
+
+**Please describe the data required**
+A clear and concise description of the data needed
+
+**Where does the data fit in the country profiles already uploaded**
+A clear and concise description of the countries this data covers and the [topics](https://takwimu.africa/) under which this data would fall
+
+**Could you share possible sources for this data if any?**
+- Link 1
+- Link 2
+- Link 3
+
+**Additional context**
+Add any other context about the data request here.
```

Comment:
Just something to think about, wouldn't having examples help or we can assume that everyone who would be making these requests is well versed into TAKWIMU?